app/src/components/inputs/inputSwitcher/inputSwitcher.jsx

```diff
@@ -20,29 +20,48 @@ import styles from './inputSwitcher.scss';
 
 const cx = classNames.bind(styles);
 
-export const InputSwitcher = ({ children, value, onChange, onFocus, onBlur, readOnly }) => {
+export const InputSwitcher = ({
+  children,
+  value,
+  onChange,
+  onFocus,
+  onBlur,
+  readOnly,
+  className,
+  labelFirst,
```

Comment:
Better to rename it to `childrenFirst` because we are using children here, not just a text message (label).